src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
+        <div id="top">
+          <h1 style="text-align: center">John Roberts</h1><br>
+          <h2 style="text-transform: uppercase;text-align: center">business expert</h2>
+          <img src="touxiang.jpg" alt="picture" style="text-align:center">
+        </div>
+
+        <div>
+          <div id="Slider">
+            <!--Slider About me-->
+            <div id="AboutMe_Container">
+              <div id="AboutMe_title">
+                <h2 style="text-transform:uppercase">About Me</h2>
+              </div>
+              <div id=AboutMe_Icon>
+                <ul>
+                  <li style="display:inline">icon1</li>
+                  <li style="display:inline">icon2</li>
+                  <li style="display:inline">icon3</li>
+                </ul>
+              </div>
+              <div id="AboutMe_description">
+                <p>paragraph1</p>
+                <p>paragraph2</p>
+              </div>
+            </div>
+            <!--Slide Experience-->
+            <div id="Experience_Container">
+              <div id="Experience_title">
+                <h2 style="text-transform:uppercase">experience</h2>
+              </div>
+              <div >
+                <button style="text-transform:uppercase">download my full Resume</button>
+              </div>
+              <div>
+                <h6>Budget Director@Wix.com</h6>
```

Comment:
From `<h2>` to `<h6>` is rather big jump.
We can style `<h3>` bit smaller if you are using `<h6>` for visual purpose.

Reply:
Actually, the company name is clickable, and always on next line, so here I suggest use another `<h3>` to wrap `<a>`